### PR TITLE
feat: add dde-shell-wrapper

### PR DESCRIPTION
**Pull request by 18202781743**
dde-shell loads org.deepin.ds.launchpad.
  e.g:
```
export DDE_SHELL_PACKAGE_PATH=/src-path/build/packages
export DDE_SHELL_PLUGIN_PATH=/src-path/build/plugins
dde-shell -p org.deepin.ds.launchpad
```

Issue: https://github.com/linuxdeepin/developer-center/issues/5810

### CODE REVIEW

**Comment by 18202781743** [comment]
https://github.com/linuxdeepin/dde-shell/pull/123

File: dde-shell-wrapper/main.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        translateDirs << (DPathBuf / appName / "translations").toString();
+    }
+    return translateDirs;
+}
+
+class LanchpadPanel : public DPanel
+{
+    Q_OBJECT
+public:
+    LanchpadPanel(QObject *parent)
```
Comment (BLumia) [inline dde-shell-wrapper/main.cpp:46]: explicit

**Review by BLumia — APPROVED** [review]

**Review by BLumia — APPROVED** [review]

**Pull request merged**